### PR TITLE
Update palo alto pan os help (#1064)

### PR DESCRIPTION
**Pull request by blaxminarayan-r7**
* [MC-689] Add information to every action is it work with Panorama or direct firewall connection

* [MC-689] Remove support header from help

### Description

Describe the proposed changes:

  -

## PR Requirements

Developers, verify you have completed the following items by checking them off:

#### Unit Tests

Review our documentation on [generating](https://docs.rapid7.com/insightconnect/unit-test-generation) and [writing](https://docs.rapid7.com/insightconnect/unit-test-primer) plugin unit tests

- [ ] Unit tests written for any new or updated code

#### In-Product Tests

If you are an InsightConnect customer or have access to an InsightConnect instance, the following in-product tests should be done:

- [ ] Screenshot of job output with the plugin changes
- [ ] Screenshot of the changed connection, actions, or triggers input within the InsightConnect workflow builder

### Style

Review the [style guide](https://docs.rapid7.com/insightconnect/style-guide/)

- [ ] For dependencies, pin [OS package](https://docs.rapid7.com/insightconnect/style-guide/#dockerfile) and [Python package](https://docs.rapid7.com/insightconnect/style-guide/#requirements.txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [ ] For error handling, use of [PluginException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations/#plugin-exceptions) and [ConnectionTestException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations#connection-exceptions)
- [ ] For logging, use [self.logger](https://docs.rapid7.com/insightconnect/sdk-guide/#logging)
- [ ] For docs, use [changelog style](https://docs.rapid7.com/insightconnect/style-guide/#changelog)
- [ ] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://docs.rapid7.com/insightconnect/style-guide/#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR
4. Add required screenshots from the In-Product Tests section

### CODE REVIEW

File: plugins/palo_alto_pan_os/help.md [hunk 1]
```diff
@@ -1069,8 +1069,33 @@ For the URL, include `https://` e.g. `https://10.0.0.1` or `https://myfirewall`.
...
+   | Remove from Policy                | Direct firewall |
+   | Retrieve Logs                     | Direct firewall |
+   | Set                               | Panorama |
+   | Set Security Policy Rule          | Direct firewall |
+   | Show                              | Panorama |
+
+
 # Version History
 
+* 6.1.4 - Add information to every action is it work with Panorama or direct firewall connection
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/help.md:1098]:
```suggestion
* 6.1.4 - Add information to every action on whether it works with Panorama or a direct firewall connection
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/add_address_object_to_group/schema.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Adds address objects to an address group"
+    DESCRIPTION = "Adds address objects to an address group. This action works with a direct connection to the firewall"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/add_address_object_to_group/schema.py:7]:
```suggestion
    DESCRIPTION = "Adds address objects to an address group. This action uses a direct connection to the firewall."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/add_external_dynamic_list/schema.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Add an external dynamic list"
+    DESCRIPTION = "Add an external dynamic list. This action works with a direct connection to the firewall"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/add_external_dynamic_list/schema.py:7]:
```suggestion
    DESCRIPTION = "Add an external dynamic list. This action uses a direct connection to the firewall."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/add_to_policy/schema.py [hunk 4]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Add a rule to a firewall security policy"
+    DESCRIPTION = "Add a rule to a firewall security policy. This action works with a direct connection to the firewall"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/add_to_policy/schema.py:7]:
```suggestion
    DESCRIPTION = "Add a rule to a firewall security policy. This action uses a direct connection to the firewall."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/check_if_address_object_in_group/schema.py [hunk 5]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Checks to see if an IP address, CIDR IP address, or domain is in an Address Group. Supports IPv6"
+    DESCRIPTION = "Checks to see if an IP address, CIDR IP address, or domain is in an Address Group. Supports IPv6. This action works with a direct connection to the firewall"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/check_if_address_object_in_group/schema.py:7]:
```suggestion
    DESCRIPTION = "Checks to see if an IP address, CIDR IP address, or domain is in an Address Group. Supports IPv6. This action uses a direct connection to the firewall."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/commit/schema.py [hunk 6]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Commits the candidate configuration"
+    DESCRIPTION = "Commits the candidate configuration. This action works with a direct connection to the firewall"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/commit/schema.py:7]:
```suggestion
    DESCRIPTION = "Commits the candidate configuration. This action uses a direct connection to the firewall."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/delete/schema.py [hunk 7]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Delete an object"
+    DESCRIPTION = "Delete an object. This action works with Panorama"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/delete/schema.py:7]:
```suggestion
    DESCRIPTION = "Delete an object. This action uses Panorama."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/edit/schema.py [hunk 8]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Edit an existing object"
+    DESCRIPTION = "Edit an existing object. This action works with Panorama"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/edit/schema.py:7]:
```suggestion
    DESCRIPTION = "Edit an existing object. This action uses Panorama."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/get/schema.py [hunk 9]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Get candidate configuration"
+    DESCRIPTION = "Get candidate configuration. This action works with Panorama"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/get/schema.py:7]:
```suggestion
    DESCRIPTION = "Get candidate configuration. This action uses Panorama."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/get_addresses_from_group/schema.py [hunk 10]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Get addresses from an address group"
+    DESCRIPTION = "Get addresses from an address group. This action works with a direct connection to the firewall"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/get_addresses_from_group/schema.py:7]:
```suggestion
    DESCRIPTION = "Get addresses from an address group. This action uses a direct connection to the firewall."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/get_policy/schema.py [hunk 11]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Get a policy by name"
+    DESCRIPTION = "Get a policy by name. This action works with a direct connection to the firewall"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/get_policy/schema.py:7]:
```suggestion
    DESCRIPTION = "Get a policy by name. This action uses a direct connection to the firewall."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/op/schema.py [hunk 12]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Runs operational command"
+    DESCRIPTION = "Runs operational command. This action works with a direct connection to the firewall"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/op/schema.py:7]:
```suggestion
    DESCRIPTION = "Runs operational command. This action uses a direct connection to the firewall."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/remove_address_object_from_group/schema.py [hunk 13]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Removes an address object from an address group. Supports IPv6"
+    DESCRIPTION = "Removes an address object from an address group. Supports IPv6. This action works with a direct connection to the firewall"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/remove_address_object_from_group/schema.py:7]:
```suggestion
    DESCRIPTION = "Removes an address object from an address group. Supports IPv6. This action uses a direct connection to the firewall."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/remove_from_policy/schema.py [hunk 14]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Remove a rule from a firewall security policy"
+    DESCRIPTION = "Remove a rule from a firewall security policy. This action works with a direct connection to the firewall"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/remove_from_policy/schema.py:7]:
```suggestion
    DESCRIPTION = "Remove a rule from a firewall security policy. This action uses a direct connection to the firewall."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/retrieve_logs/schema.py [hunk 15]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Queries firewall logs"
+    DESCRIPTION = "Queries firewall logs. This action works with a direct connection to the firewall"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/retrieve_logs/schema.py:7]:
```suggestion
    DESCRIPTION = "Queries firewall logs. This action uses a direct connection to the firewall."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/set/schema.py [hunk 16]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Create a new object"
+    DESCRIPTION = "Create a new object. This action works with Panorama"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/set/schema.py:7]:
```suggestion
    DESCRIPTION = "Create a new object. This action uses Panorama."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/set_address_object/schema.py [hunk 17]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Create a new address object. Supports IPv6"
+    DESCRIPTION = "Create a new address object. Supports IPv6. This action works with a direct connection to the firewall"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/set_address_object/schema.py:7]:
```suggestion
    DESCRIPTION = "Create a new address object. Supports IPv6. This action uses a direct connection to the firewall."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/set_security_policy_rule/schema.py [hunk 18]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Creates a new Security Policy Rule"
+    DESCRIPTION = "Creates a new Security Policy Rule. This action works with a direct connection to the firewall"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/set_security_policy_rule/schema.py:7]:
```suggestion
    DESCRIPTION = "Creates a new Security Policy Rule. This action uses a direct connection to the firewall."
```

File: plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/show/schema.py [hunk 19]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Gets active configuration"
+    DESCRIPTION = "Gets active configuration. This action works with Panorama"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/komand_palo_alto_pan_os/actions/show/schema.py:7]:
```suggestion
    DESCRIPTION = "Gets active configuration. This action uses Panorama."
```

File: plugins/palo_alto_pan_os/plugin.spec.yaml [hunk 20]
```diff
@@ -949,7 +949,7 @@ actions:
         required: true
   get_addresses_from_group:
     title: Get Addresses from Group
-    description: Get addresses from an address group
+    description: Get addresses from an address group. This action works with a direct connection to the firewall
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/plugin.spec.yaml:952]: For each action, please rephrase and punctuate the description. Please replace `works with` with `uses`, and add `.` to the end of each sentence.

File: plugins/palo_alto_pan_os/setup.py [hunk 21]
```diff
@@ -2,10 +2,10 @@
 from setuptools import setup, find_packages
 
 
-setup(name="palo_alto_pan_os-rapid7-plugin",
-      version="6.1.3",
+setup(name="palo_alto_pan_os-bindu-plugin",
+      version="0.1.0",
       description="Manage Palo Alto Networks firewall devices",
-      author="rapid7",
+      author="bindu",
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/setup.py]:
```suggestion
      author="rapid7",
```

File: plugins/palo_alto_pan_os/setup.py [hunk 22]
```diff
@@ -2,10 +2,10 @@
 from setuptools import setup, find_packages
 
 
-setup(name="palo_alto_pan_os-rapid7-plugin",
-      version="6.1.3",
+setup(name="palo_alto_pan_os-bindu-plugin",
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/setup.py]:
```suggestion
setup(name="palo_alto_pan_os-rapid7-plugin",
```

File: plugins/palo_alto_pan_os/bin/komand_palo_alto_pan_os [hunk 23]
```diff
@@ -5,8 +5,8 @@ import json
 from sys import argv
 
 Name = "Palo Alto Firewall"
-Vendor = "rapid7"
-Version = "6.1.3"
+Vendor = "bindu"
```
Comment (mberezin-r7) [inline plugins/palo_alto_pan_os/bin/komand_palo_alto_pan_os]:
```suggestion
Vendor = "rapid7"
```

File: plugins/palo_alto_pan_os/help.md (hunk 1 above)
Comment (blaxminarayan-r7) [reply plugins/palo_alto_pan_os/help.md]: Fixed in latest commit

File: plugins/palo_alto_pan_os/setup.py (hunk 22 above)
Comment (blaxminarayan-r7) [reply plugins/palo_alto_pan_os/setup.py]: Fixed in latest commit. Regenerated the checksum and other files

File: plugins/palo_alto_pan_os/bin/komand_palo_alto_pan_os (hunk 23 above)
Comment (blaxminarayan-r7) [reply plugins/palo_alto_pan_os/bin/komand_palo_alto_pan_os]: Fixed in latest commit, regenerated.

**Review by mberezin-r7 — APPROVED** [review]

**Review by mberezin-r7 — APPROVED** [review]

**Pull request merged**